src/entities/Proposal/model.ts

```diff
@@ -162,4 +162,15 @@ export default class ProposalModel extends Model<ProposalAttributes> {
 
     return proposals.map(this.parse)
   }
-}
\ No newline at end of file
+
+  static async getTitle(proposal_id: string) {
```

Comment:
Is this necessary? I think you can do the query like this:

`const proposal = await ProposalModel.findOne<ProposalAttributes>({ id, deleted: false })`